src/main/java/Duke.java

```diff
@@ -1,10 +1,76 @@
+import java.util.Objects;
+import java.util.Scanner;
+
 public class Duke {
+    public static String concatenateChar(String line, String start, String end) {
+        int indexOfStart = line.indexOf(start) + 1;
+        int indexOfEnd = 0;
+        if (end == null) {
+            indexOfEnd = line.length();
+        } else {
+            indexOfEnd = line.indexOf(end);
+        }
+        String taskName = line.substring(indexOfStart, indexOfEnd);
+        return taskName;
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
+        String logo = " _           _        \n"
```

Comment:
```suggestion
        private static final String LOGO = " _           _        \n"
```
Perhaps you can capitalise the variable name since it is a constant